src/app/not-found.tsx

```diff
@@ -13,7 +13,8 @@ export default function Page() {
     name: "graphql.github.io",
   }
 
-  const title = `Found broken \`${mounted ? pathname?.replace(/\/$/, "") : ""}\` link. Please fix!`
+  const referrer = document?.referrer
```

Comment:
```suggestion
  const referrer = mounted && document.referrer
```
more safe (document doesn't exist on server